addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -194,6 +195,85 @@ def format_alphanumeric(text_to_convert):
             if rc_refund:
                 document_total = -abs(document_total)
 
+        # List of dictionaries passed to the template for the part of 'DettaglioLinee'
```

Comment:
Can we make this a function that takes invoice_lines and fixes them?
Like with `is_downpayment` and `rc_refund` as arguments 
We need a bit of readability here.
People keep on asking for hooks and modularity, and I also think that this function is really really too big